utils/command_line_params.cpp

```diff
@@ -55,8 +55,9 @@ quda::mgarray<char[256]> mg_vec_infile;
 quda::mgarray<char[256]> mg_vec_outfile;
 QudaInverterType inv_type;
 bool inv_deflate = false;
+bool inv_multigrid = false;
 QudaInverterType precon_type = QUDA_INVALID_INVERTER;
-int multishift = 0;
+int multishift = 1;
```

Comment:
Why has this been set to 1 from the default of 0?

Reply:
I used `multishift` as a parameter describing the number of masses. If set to one, the invert_test goes through the single mass code path, `invertQuda()`. If greater than one, multiple masses are created (as a function of the base mass) and `invertMultiShiftQuda()` is called. I suppose `multishift` in its current context means `masses`.

Reply:
Ah ok, that makes sense.  Can you add a comment explaining this?